### DISCUSSION TITLE
What to do with retired licenses?

### DISCUSSION THREAD

**Discussion by krysal** [WordPress/openverse#71] [Q&A]
Working on the audio results page with the sample in the local API, I found some interesting results:

![CleanShot 2021-09-17 at 15 05 43@2x](https://user-images.githubusercontent.com/9145885/133841168-a9e119e8-811a-4974-9195-cc5ede2da3d1.png)

There are some audio tracks associated with _retired_ Creative Commons (CC) licenses ([nc-sampling+][nc-plus], [sampling+][sp-plus]), this means that CC no longer recommends their use, but these are still valid CC licenses, so several questions come to my mind:
- Should we include these results in the Catalog or ignore them? They passed the license validation.
- If we include them, should we add them to the licenses filter? What is the compatibility with the commercial or modification use? With time I guess the number of results for these will be low, and potentially there could be several legacy licenses, and my guess is that the CC only wanted to cover the current ones.
- Do we include them in the results but not in filters? Only for showing icons, name, version, etc. 

[nc-plus]: https://creativecommons.org/licenses/nc-sampling+/1.0/
[sp-plus]: https://creativecommons.org/licenses/sampling+/1.0/

**Comment by dhruvkb** [comment]
These are good questions! There are three bits that haven't accounted for:
- Sampling
- Sampling Plus
- Remix

**Comment by obulat** [comment]
I prefer the last option: include the licenses with their names and icons in the results, but not in the filters. This would ensure that all openly-licensed content is surfaced in our search. At the same time we will not be featuring no-longer-recommended licenses prominently. And for that reason (as well as simplicity) I would not add them to non-commercial or modification-allowed license type filters.

**Comment by zackkrida** [comment]
It makes sense to me to show these icons on single results and on results in the list view, but to exclude these licenses from the filters.

**Comment by sarayourfriend** [comment]
Showing the icons and including them in the results seems good to me. But why exclude them from the filters? It might be useful for some people doing CC metadata research to be able to access that data in some way through the UI or even the API 🤔

**Comment by fcoveram** [comment]
My main concern here is hiding content that was intentionally marked to be visible under a specific context, and Openverse not displaying it because it is not compatible with current/new legal tools. That is why I was looking for recommendations on displaying information under retired licenses and just found the arguments of why the decision. We might ask on CC slack channels. 

On the other hand, as @sarayourfriend pointed, I can not find a reason for why not showing them as filters. What if we add a category called **Retired licenses**? Thinking of the feature that enables/disables a license when selecting an **Use** criteria.

Regarding displaying the license information on the audio component, I am drawn to showing license icons on the search result page and the deed information of the license with a warning message (similar to what [CC does](https://creativecommons.org/licenses/sampling+/1.0/deed)) on the single result page.

**Comment by zackkrida** [comment]
Regarding the motivations not to show these licenses in the filters:

- There are *many* deprecated licenses and they add a lot of visual clutter
- At Creative Commons, the use of those licenses is discouraged and not something they wanted to showcase

I'd be fine including these in the filters but hiding them by default with a small 'more' link below the checkboxes that expands the list or something like that.
